src/Model/CalendarEventsModel.php

```diff
@@ -17,6 +17,9 @@ class CalendarEventsModel extends \Contao\CalendarEventsModel
 {
     /**
      * Checks a given event has sub events.
+     *
+     * @deprecated Deprecated since 1.7.0, to be removed in 2.0.
+     *             Use EventsManager::hasSubEvents instead.
      */
     public static function hasSubEvents(int $event): bool
     {
```

Comment:
Please route the deprecated functions to the newly created ones.

Reply:
With 'route' you mean the full namespace of EventsManager?

Reply:
@AlexejKossmann No, @Defcon0 means you should use the new methods here (`return System::getContainer()->get(Manager)->hasSubEvents();`).